samples/matter/common/src/bridge/ble_connectivity_manager.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include "ble_bridged_device.h"
+#include "bridged_device_data_provider.h"
+
+#include <zephyr/bluetooth/addr.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/kernel.h>
+
+class BLEConnectivityManager {
```

Comment:
Looks like `struct` keyword is populated across the `BLEConnectivityManager`, so I suggest to remove all redundant occurrences simply to have less code overall.